### ISSUE TITLE
Methanol processing 2

### ISSUE THREAD

**Issue by ShadowGlass0** [pyanodon/pycoalprocessing#145]
Methanol processing 2 and 1 are both green science. I generally don't like this sort of thing. If the recipes in the 2nd tech are upgrades (they are) then the recipes from the first tech will never be used.
In this case Methanol processing 2 can be changed to require chemical science. Energy gen 2 is dependent on Methanol proc 2, but it doesn't have to be. Methanol proc 1 already has a recipe for RefSyngas, which is why the dependency is there.

In summary:
- Add chemical science to Methanol processing 2 (+prerequisite to Chemical science pack tech)
- Change Energy generation 2 prerequisite from Methanol processing 2 to Methanol processing 1
- Move Methanol combustion mixture recipe from Methanol processing 2 to Energy generation 2

**Comment by kingarthur91** [comment]
give that commit a look and if everything is ok close this

**Comment by ShadowGlass0** [comment]
Redundant prerequisites:
- Methanol processing 1 (already required for Chemical science pack)
- Mukmoux (already required for Chemical science pack)
